src/home_robot_hw/home_robot_hw/stretch_client/modules/manip.py

```diff
@@ -186,6 +186,7 @@ def move_gripper(self, target, blocking: bool = True):
         }
         self._ros_client.send_trajectory_goals(joint_goals)
 
+        self._register_wait(self._ros_client.wait_for_trajectory_action)
```

Comment:
is wait for trajectory action the best way to do this? should we be waiting for an error term instead?